pkg/controller/certificates/controller.go

```diff
@@ -181,6 +187,51 @@ func (cc *CertificateController) maybeSignCertificate(key string) error {
 	}
 	csr := obj.(*certificates.CertificateSigningRequest)
 
+	func() {
+		if !cc.approveAllKubeletCSRs ||
```

Comment:
I think instead of 
```golang
		if !cc.approveAllKubeletCSRs ||
			IsCertificateRequestApproved(csr) ||
			!IsCertificateRequestDenied(csr) {
```
we want 
```golang
		if !cc.approveAllKubeletCSRs ||
			!IsCertificateRequestApproved(csr) ||
			IsCertificateRequestDenied(csr) {
```